include/deal.II/base/discrete_time.h

```diff
@@ -103,11 +114,12 @@ DEAL_II_NAMESPACE_OPEN
  *   size $t_{n+1} - t_n$ and *previous* time step size $t_n - t_{n-1}$. In
  *   this stage, it is a perfect occasion to generate text output using print
  *   commands within the user's code. Additionally, post-processed outputs can
- *   be prepared here which can be viewed later by visualization programs such
- *   as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the snapshot
- *   stage, the code can assess the quality of the previous step and decide
- *   whether it wants to increase or decrease the time step size. The step
- *   size for the next time step can be modified here.
+ *   be prepared here, which can then later be viewed by visualization programs
+ *   such as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the
+ * snapshot stage, the code can assess the quality of the previous step and
+ * decide whether it wants to increase or decrease the time step size. The step
```

Comment:
For consistency, do you want to indent these two lines with the previous ones, i.e., add two more spaces?

Reply:
Yes, done. An effect of having too long a line that clang-format broke and I didn't check.